__init__.py

```diff
@@ -91,8 +91,9 @@ def initialize(self):
 
         # Invoke update_display in one second to allow it to disable the
         # cancel intent, since there are no timers to cancel yet!
-        self.schedule_repeating_event(self.update_display,
-                                      None, 1, name='ShowTimer')
+        if self.platform != "mycroft_mark_2":
```

Comment:
if i understand correctly this is just meant for mark1, can we do if platform == mark1  instead? as commented bellow the GUI is not mark2 exclusive, eg, if i have platform == ovos i still want the QT GUI to work properly

Reply:
Will move this to self.gui.connected() check, as there might be more headless devices that want to use the general timer path apart from mark 1